lib/constants.js

```diff
@@ -7,6 +7,7 @@ REOL_CHAN_ID  = '296232884623376384'
 ANNOUNCEMENTS_ID = '413504832356352020'
 PHOTO_GALLERY_ID = '423787873448493056'
 DISCOGRAPHY_ID = '515055814755614730'
+COLORS = ["Red", "Blue", "Green", "Purple", "Yellow", "Orange", "Pink"]
```

Comment:
it might be fun to not have hardcoded colours and allow users to assign their own hex for their custom role, to spice things up a bit. That could be your next task 😄

Reply:
@0xbbadbeef the roles we have on the server are region-based instead of colors, so that we can easily see who is from which region

Reply:
@causztic ahh, okie

Reply:
@0xbbadbeef ok that'll be another PR, but yeah colors are tied to roles so for hexes for custom role will not be possible with the current discord API because that would require new colors being created on the fly..and there are limited roles I can create on the server.